### PR TITLE
add ADR wrt OWNERS file

### PR DESCRIPTION
**Pull request by goern**
This PR provides an ADR to get a clear structure for OWNERS and teams

Signed-off-by: Christoph Görn <goern@redhat.com>

/kind feature

### CODE REVIEW

File: adr/0000-use-OWNERS-rather-than-github-teams.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+the roles within each repository and/or within the organization.
+
+## Decision Drivers <!-- optional -->
+
+* clarity and easy tracability
+
+## Considered Options
+
+* OWNERS-teams-mix - we can use the OWNERS file in each repository to declare reviews/approver roles by referencing a GitHub team
+* OWNERS-only - we can use the OWNERS file to explicitly declare the roles
```
Comment (tumido) [inline adr/0000-use-OWNERS-rather-than-github-teams.md:20]: Hm.. I think this option is a bit more complex than outlined here. We also use [`OWNERS_ALIASES`](https://github.com/open-services-group/community/blob/main/OWNERS_ALIASES) - as a soft drop in replacement for teams.

Comment (harshad16) [reply adr/0000-use-OWNERS-rather-than-github-teams.md:20]: I agree on this point, we should address having the OWNERS_ALIASES.

Also we should state, folks who are needed to be admin of repo(permission to change setting) should be defined in the github team and addressed with OWNERS with team.
i don't know if only stating a person as owner in OWNERS file or OWNERS_ALIASES grant them admin privileges in that repo, as my understanding it , only prow think that they are good to be maintainers and can approve pr or issues.

Comment (goern) [reply adr/0000-use-OWNERS-rather-than-github-teams.md:20]: ... but even in the kubernetes universe, the initial commit most often comes from a human.

**Review by durandom — COMMENTED** [review]
It would be cool if we could address GitHub teams in issues. 
IIRC periobolos can also manage GH teams? If so, can it map the `OWNERS_ALIAS` to teams?
I would also argue for having all group/team memberships in a central location instead of it being spread across repositories. 

repo/OWNERS - references team 
community/OWNERS_ALIASES - manages team membership

**Review by harshad16 — COMMENTED** [review]
/lgtm
:100:

**Review by tumido — COMMENTED** [review]
/lgtm

**Pull request merged**